zipkin-server/src/main/java/zipkin/server/ZipkinServer.java

```diff
@@ -49,6 +49,8 @@ public static void main(String[] args) {
     new SpringApplicationBuilder(ZipkinServer.class)
       .banner(new ZipkinBanner())
       .initializers(new ZipkinModuleImporter(), new ZipkinActuatorImporter())
+      // Avoids potentially expensive ns lookup and inaccurate startup timing
```

Comment:
ns -> DNS

My first reading was this is referring to System.nanoTime and thought hrm?